test/unit/utilities/test_common_functions.py

```diff
@@ -180,10 +180,25 @@ def test_download_test_files_hydraulic(self):
         # delete if already exists
         if testmodels_path.exists():
             cf.rm_tree(testmodels_path)
-        cf.download_test_resources('hydraulic')
+        cf.download_test_resources('hydraulic', with_regression=True)
         ifc_path = testmodels_path / 'ifc'
         regression_path = testmodels_path / 'regression_results'
-        # TODO #539: include hvac regression results here when implemented
+        for path in [ifc_path, regression_path]:
+            if not path.exists():
+                raise AssertionError(
+                    f"Path does not exist: {path}, download of "
+                    f"architecture IFC files didn't work.")
+
+    def test_download_test_files_mixed(self):
+        testmodels_path = Path(__file__).parent.parent.parent / \
+                          'resources/mixed'
+        # delete if already exists
+        if testmodels_path.exists():
+            cf.rm_tree(testmodels_path)
+        cf.download_test_resources('mixed', with_regression=False)
+        ifc_path = testmodels_path / 'ifc'
+        # TODO #1 include regression results for mixed  when they exist
+        regression_path = testmodels_path / 'regression_results'
         for path in [ifc_path]:
             if not path.exists():
                 raise AssertionError(
```

Comment:
AssertionError should be `f"Path does not exist: {path}, download of mixed IFC files didn't work.")`